packages/react-components/src/components/utils/chatMessageContentGenerators.tsx

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { _formatString } from '@internal/acs-ui-common';
+import { Parser } from 'html-to-react';
+import Linkify from 'react-linkify';
+import { ChatMessage } from '../../types/ChatMessage';
+import { LiveMessage } from 'react-aria-live';
+import { Link } from '@fluentui/react';
+
+/** @private */
+export const GenerateMessageContent = (message: ChatMessage, liveAuthorIntro: string): JSX.Element => {
```

Comment:
Can this just be `MessageContent` or `FormattedMessageContent` (and accordingly rename the file)?

Reply:
Updated to a react component `<ChatMessageContent />`